webapp-django/crashstats/supersearch/models.py

```diff
@@ -232,15 +233,15 @@ def get(self, **kwargs):
 
 class SuperSearchFields(ESSocorroMiddleware):
 
-    # Read it in once as a class attribute since it'll never change
-    # unless the .json file on disk changes and if that happens you
-    # will have reloaded the Python process.
+    # Read it in once as a class attribute since it'll never change unless the
+    # Python code changes and if that happens you will have reloaded the
+    # Python process.
     _fields = SuperSearchFieldsWithoutConfig().get()
 
     API_WHITELIST = None
 
     def get(self):
-        return self._fields
+        return copy.deepcopy(self._fields)
```

Comment:
This isn't a huge datastructure, but there's always a penalty for a deepcopy. I don't know how big a difference it'll make. We should notice it in New Relic.

If it turns out to be a big problem, then someone can look at fixing the users of the data to copy before adjusting and we can remove this copy. Or something along those lines.